scripts/core/datetime/datetime.ts

```diff
@@ -2,6 +2,7 @@ import _ from 'lodash';
 import {gettext} from 'core/utils';
 import moment from 'moment-timezone';
 import {appConfig} from 'appConfig';
+import {labels as timezoneLabels} from 'apps/dashboard/world-clock/timezones-all-labels';
```

Comment:
why not export it as `timezoneLabels`? It's easier to find usages when the name is the same as in the exporting file.